src/components/homepage/UpEvents.astro

```diff
@@ -0,0 +1,70 @@
+---
+const eventsData = [
+  {
```

Comment:
@axyut use json or some config file to get list of events

Reply:
working on it